chase/frontend/app/page.tsx

```diff
@@ -1,11 +1,85 @@
-import React from "react";
+"use client";
+import React, { Suspense, useState, useRef } from "react";
+import Image from "next/image";
+import { Button } from "primereact/button";
+import { Toast } from "primereact/toast";
+import Loading from "./loading";
 
 export default function Home() {
+  const [loadingParticipants, setLoadingParticipants] = useState(false);
+  const [loadingChairs, setLoadingChairs] = useState(false);
+
+  const linkRedirectingToast = useRef(null);
+
+  const showLinkRedirectingToast = () => {
+    linkRedirectingToast.current.show({
+      severity: "info",
+      summary: "Redirecting...",
+      detail: "You will be redirected to the login page.",
+    });
+  };
+
+  function loadParticipants() {
+    setLoadingParticipants(true);
+
+    // show toast
+    showLinkRedirectingToast();
+
+    // navigate to the participant login page
+    setTimeout(() => {
+      window.location.href = "/login/participant";
+    }, 1000);
```

Comment:
Kk, thats what I thought. But I wanted to mess around with the possibilites a little bit and showcase them (also to myself). But I agree it is not a good Idea to keep it since we might forget about it.